core/Command/App/ListApps.php

```diff
@@ -112,14 +112,27 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		if ($input->getOption('enabled') || $neitherSpecified) {
 			\sort($enabledApps);
 			foreach ($enabledApps as $app) {
-				$apps['enabled'][$app] = (isset($versions[$app])) ? $versions[$app] : true;
+				$appDetailRecord = [];
+
+				if (isset($versions[$app])) {
+					$appDetailRecord['Version'] = $versions[$app];
+				}
+
+				$appDetailRecord['Path'] = \OC_App::getAppPath($app);
+				$apps['enabled'][$app] =$appDetailRecord;
```

Comment:
```suggestion
				$apps['enabled'][$app] = $appDetailRecord;
```
That is odd that code-style does not enforce consistent spaces around `=`